frontend/tsconfig.json

```diff
@@ -24,25 +20,17 @@
 		"noEmit": true,
 		"baseUrl": "./src",
 		"paths": {
-			"@constants/*": [
-				"/container/OnboardingContainer/constants/*"
-			]
+			"@constants/*": ["/container/OnboardingContainer/constants/*"]
 		},
 		"downlevelIteration": true,
 		"plugins": [
 			{
 				"name": "typescript-plugin-css-modules"
 			}
 		],
-		"types": [
-			"node",
-			"jest"
-		],
+		"types": ["node", "jest"]
 	},
-	"exclude": [
-		"node_modules",
-		"./src/container/OnboardingContainer/constants/*.ts"
-	],
+	"exclude": ["node_modules"],
```

Comment:
we do not need to exclude the constants files here, we have type declarations for the same